src/Symfony/Bundle/FrameworkBundle/KernelBrowser.php

```diff
@@ -104,6 +107,31 @@ public function enableReboot()
         $this->reboot = true;
     }
 
+    /**
+     * @param UserInterface $user
+     */
+    public function loginUser($user, string $firewallContext = 'main'): self
```

Comment:
Removing the typehint allowed me to have a bit nicer error message when Security is not installed.